frontend/packages/integration-tests-cypress/tests/app/admission-webhook-warning-notifications.cy.ts

```diff
@@ -0,0 +1,172 @@
+    name: DEPLOY_NAME,
+    warning: WARNING_BAR,
+    resource: 'deployments',
+    path: 'apis/apps',
+  },
+];
+const pod1ReqObj = `apiVersion: v1
+kind: Pod
+metadata:
+  name: ${POD_NAME}-a
+  labels:
+    app: httpd
+  namespace: ${testName}
+spec:
+  securityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+  containers:
+    - name:  ${CONTAINER_NAME}
+      image: 'image-registry.openshift-image-registry.svc:5000/openshift/httpd:latest'
+      ports:
+        - containerPort: 8080
+      securityContext:
+        allowPrivilegeEscalation: false
+        capabilities:
+          drop:
+            - ALL`;
+
+const bulkResourcesReqObj = `apiVersion: v1
+kind: Pod
+metadata:
+  name: ${POD_NAME}-b
+  labels:
+    app: httpd
+  namespace: ${testName}
+spec:
+  securityContext:
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+  containers:
+    - name:  ${CONTAINER_NAME}
+      image: 'image-registry.openshift-image-registry.svc:5000/openshift/httpd:latest'
+      ports:
+        - containerPort: 8080
+      securityContext:
+        allowPrivilegeEscalation: false
+        capabilities:
+          drop:
+            - ALL
+---           
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ${DEPLOY_NAME}
+  annotations: {}
+  namespace: ${testName}
+spec:
+  selector:
+    matchLabels:
+      app: deploy1
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: deploy1
+    spec:
+      containers:
+        - name: ${CONTAINER_NAME}
+          image: >-
+            image-registry.openshift-image-registry.svc:5000/openshift/httpd:latest
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+          env:
+            - name: app
+              value: frontennd
+      imagePullSecrets: []
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+  paused: false
+`;
+
+describe('Admission Webhook warning notification', () => {
+  before(() => {
+    cy.login();
+    cy.createProjectWithCLI(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.visit('/');
+    cy.deleteProjectWithCLI(testName);
+  });
+
+  it('Create a pod and display Admission Webhook warning notification', () => {
+    cy.visit(`/k8s/ns/${testName}/import`);
+    yamlEditor.isImportLoaded();
+    yamlEditor.setEditorContent(pod1ReqObj).then(() => {
+      cy.intercept('POST', `/api/kubernetes/api/v1/namespaces/${testName}/pods`, (req) => {
+        req.continue((res) => {
+          res.headers = {
+            Warning: WARNING_FOO,
+          };
+        });
+      }).as(POD_CREATED_ALIAS);
+      yamlEditor.clickSaveCreateButton();
+      cy.wait(`@${POD_CREATED_ALIAS}`, WAIT_OPTION);
+      detailsPage.sectionHeaderShouldExist('Pod details');
+      cy.byTestID(LEARN_MORE_ID).parents().contains('Admission Webhook Warning');
+      cy.byTestID(LEARN_MORE_ID)
```

Comment:
Use [data-test](https://github.com/openshift/console/blob/a4e560ac5d50c94fc36b351d109bc1db63968121/frontend/packages/console-shared/src/components/toast/ToastProvider.tsx#L61) on the `<Alert>` itself.